docs/create-custom-image-using-existing-docker-image.md

```diff
@@ -171,18 +221,26 @@ docker push customrepository/custom-alfresco-repository:customTag
 ### Apply AMPs to the latest Share image (at this moment).
 FROM alfresco/alfresco-share:6.0.0-rc2
 
-### Copy the keystore build context to the appropriate location for your application server
-COPY keystore /usr/local/tomcat/keystore
+# Need these viariables to give permissions to tomcat (refer Dockerfile below)
+ARG GROUPNAME=Alfresco
+ARG USERNAME=alfresco
+ARG TOMCAT_DIR=/usr/local/tomcat
+
+# Need to switch to root to add the new files
```

Comment:
Please correct to:

Alfresco user does not have permissions to modify webapps or configuration. Switch to root.
The access will be fixed after all operations are done.